batch/src/main/java/org/openmrs/analytics/JdbcFetchUtil.java

```diff
@@ -148,14 +148,12 @@ public void processElement(@Element KV<String, Iterable<String>> element,
 	public Integer fetchMaxId(String tableName) throws SQLException {
 		String tableId = tableName + "_id";
 		Connection con = this.comboPooledDataSource.getConnection();
-		Statement statement = con.createStatement();
-		ResultSet resultSet = statement
-		        .executeQuery(String.format("SELECT MAX(`%s`) as max_id FROM %s", tableId, tableName));
-		resultSet.first();
-		int maxId = resultSet.getInt("max_id");
-		resultSet.close();
```

Comment:
Any reason why we are deleting the closing of the statement and the resultset? Do we want to close them at the `finally` block of the try catch?